testing/src/main/java/org/oppia/testing/FakeCrashLogger.kt

```diff
@@ -0,0 +1,21 @@
+package org.oppia.testing
+
+import java.lang.Exception
+import org.oppia.testing.TestFirebaseModule.Companion.exceptionList
+import org.oppia.util.firebase.CrashLogger
+
+class FakeCrashLogger : CrashLogger {
+
+  override fun logException(exception: Exception) {
+    exceptionList.add(exception)
+  }
+
+  fun getMostRecentException(): Exception {
+    val size = exceptionList.size
+    return exceptionList[size - 1]
```

Comment:
You probably want to check if size is > 0 here before returning

Reply:
Done

Reply:
This would return -1 if the size = 0, you should check if size > 0